README.md

```diff
@@ -2,27 +2,28 @@
 
 ## About
 
-The sambacc is an young project that aims to consolidate and coordinate
-configuration of [Samba](http://samba.org), and related components, when
-running in a container. The configuration of one or many instances can be
-provided to the tool by way of a JSON file which then handles the low level
-details of actually configuring smbd and other elements in the container
+The sambacc project aims to consolidate and coordinate configuration of
+[Samba](http://samba.org), and related components, when running in a
+container. The configuration of one or many server instances can be
+managed by the tool with the use of configuration files.  These
+configuration files act as a superset of the well-known smb.conf, to
+configure Samba, as well as other low level details of the container
 environment.
 
 
 ## Rationale
 
-Samba is a powerful and unique tool for implementing the SMB protocol and a
-software stack to support it on unix-like systems. However, it's potentially
-challenging to set up and manage many instances of Samba by-hand, especially
-when running under a container orchestration system.
+Samba is a powerful and unique tool for implementing the SMB protocol and
+a software stack to support it on unix-like systems. However, it's
+potentially challenging to set up and manage many instances of Samba
+by-hand, especially when running under a container orchestration system.
 
-The idea behind sambacc is mainly to automate all of the low level steps of
-setting up smbd, users, groups, and other supporting components. The tool is
-also designed to consume one "site-wide" configuration file that can be
-maintained across many container instances. sambacc is written in Python as
-samba provides Python bindings for some of the aspects of samba we need to
-control.
+The idea behind sambacc is to automate much of the low level steps needed
+to set up smbd or other samba servers, users, groups, and other supporting
```

Comment:
```suggestion
to set up samba servers, users, groups, and other supporting
```

Reply:
Fixed with a tweaked version, "daemons" instead of "servers" to imply daemon processes like smbd rather than an abstract server.